internal/wrphandlers/qos/priority_queue.go

```diff
@@ -15,6 +14,12 @@ import (
 
 var ErrMaxMessageBytes = errors.New("wrp message payload exceeds maxMessageBytes")
 
+var (
+	// https://xmidt.io/docs/wrp/basics/#request-delivery-response-rdr-codes
+	messageIsTooLarge                int64 = 4
```

Comment:
Instead of making these globals so they can be returned as a pointer, I'd make them a const an where used create a local variable & return the pointer.

A mistake where an int is written to will change the value forever is my concern.